### PR TITLE
feat: enable UnUniFi module

### PR DESCRIPTION
**Pull request by Senna46**
Enables the original UnUniFi module.

### CODE REVIEW

**Comment by Senna46** [comment]
Inclusion of epoch module will cause an error. Disable it once.
The cause of this error is that it is trying to decode an empty byte array and failing.
```
CONSENSUS FAILURE!!! module=consensus err="UnmarshalJSON cannot decode empty bytes" stack="goroutine 48 [running]:\nruntime/debug.Stack()\n\t/usr/local/go/src/runtime/debug/stack.go:24 +0x7a\ngithub.com/cometbft/cometbft/consensus.(*State).receiveRoutine.func2()\n\t/home/user/go/pkg/mod/github.com/cometbft/cometbft@v0.37.0/consensus/state.go:732 +0x5e\npanic({0x3a65820, 0xc0012cf180})\n\t/usr/local/go/src/runtime/panic.go:890 +0x267\ngithub.com/cosmos/cosmos-sdk/x/params/types.Subspace.Get({{0x4ab2200, 0xc0001585b0}, 0xc00011c610, {0x4a7bb28, 0xc0012a2530}, {0x4a7bb78, 0xc0012a26d0}, {0xc000d3a990, 0x8, 0x28}, ...}, ...)\n\t/home/user/go/pkg/mod/github.com/ununifi/cosmos-sdk@v0.47.1-bank-rc2/x/params/types/subspace.go:110 +0x1df\ngithub.com/UnUniFi/chain/x/stakeibc/keeper.(*Keeper).GetParam(_, {{0x4a9e658, 0xc000056100}, {0x4ab2eb0, 0xc00004d4c0}, {{0xb, 0x0}, {0xc000bdfdf0, 0x4}, 0x1, ...}, ...}, ...)\n\t/home/user/github.com/UnUniFi/chain/x/stakeibc/keeper/params.go:22 +0x145\ngithub.com/UnUniFi/chain/x/stakeibc/keeper.Keeper.BeforeEpochStart({{0x4ab2200, 0xc0001585b0}, {0x4a7bb28, 0xc0012a2600}, {0x4a7bb28, 0x0}, {{0x4ab2200, 0xc0001585b0}, 0xc00011c610, {0x4a7bb28, ...}, ...}, ...}, ...)\n\t/home/user/github.com/UnUniFi/chain/x/stakeibc/keeper/hooks.go:94 +0x1465\ngithub.com/UnUniFi/chain/x/stakeibc/keeper.Hooks.BeforeEpochStart({{{0x4ab2200, 0xc0001585b0}, {0x4a7bb28, 0xc0012a2600}, {0x4a7bb28, 0x0}, {{0x4ab2200, 0xc0001585b0}, 0xc00011c610, {0x4a7bb28, ...}, ...}, ...}}, ...)\n\t/home/user/github.com/UnUniFi/chain/x/stakeibc/keeper/hooks.go:194 +0x9b\ngithub.com/UnUniFi/chain/x/epochs/types.MultiEpochHooks.BeforeEpochStart({_, _, _}, {{0x4a9e658, 0xc000056100}, {0x4ab2eb0, 0xc00004d4c0}, {{0xb, 0x0}, {0xc000bdfdf0, ...}, ...}, ...}, ...)\n\t/home/user/github.com/UnUniFi/chain/x/epochs/types/hooks.go:33 +0xfd\ngithub.com/UnUniFi/chain/x/epochs/keeper.Keeper.BeforeEpochStart({{_, _}, {_, _}, {_, _}}, {{0x4a9e658, 0xc000056100}, {0x4ab2eb0, 0xc00004d4c0}, ...}, ...)\n\t/home/user/github.com/UnUniFi/chain/x/epochs/keeper/hooks.go:16 +0xbb\ngithub.com/UnUniFi/chain/x/epochs/keeper.Keeper.BeginBlocker.func1(0x0, {{0xc0012c1360, 0xa}, {0x397b797c, 0xedbefe8f7, 0x0}, 0x13a52453c000, 0x1, {0x397b797c, 0xedbefe8f7, ...}, ...})\n\t/home/user/github.com/UnUniFi/chain/x/epochs/keeper/abci.go:47 +0x8a5\ngithub.com/UnUniFi/chain/x/epochs/keeper.Keeper.IterateEpochInfo({{_, _}, {_, _}, {_, _}}, {{0x4a9e658, 0xc000056100}, {0x4ab2eb0, 0xc00004d4c0}, ...}, ...)\n\t/home/user/github.com/UnUniFi/chain/x/epochs/keeper/epoch.go:49 +0x378\ngithub.com/UnUniFi/chain/x/epochs/keeper.Keeper.BeginBlocker({{_, _}, {_, _}, {_, _}}, {{0x4a9e658, 0xc000056100}, {0x4ab2eb0, 0xc00004d4c0}, ...})\n\t/home/user/github.com/UnUniFi/chain/x/epochs/keeper/abci.go:17 +0x2a8\ngithub.com/UnUniFi/chain/x/epochs.AppModule.BeginBlock({{{_, _}}, {{_, _}, {_, _}, {_, _}}}, {{0x4a9e658, 0xc000056100}, ...}, ...)\n\t/home/user/github.com/UnUniFi/chain/x/epochs/module.go:147 +0xc5\ngithub.com/cosmos/cosmos-sdk/types/module.(*Manager).BeginBlock(_, {{0x4a9e658, 0xc000056100}, {0x4ab2eb0, 0xc00004d4c0}, {{0xb, 0x0}, {0xc000bdfdf0, 0x4}, 0x1, ...}, ...}, ...)\n\t/home/user/go/pkg/mod/github.com/ununifi/cosmos-sdk@v0.47.1-bank-rc2/types/module/module.go:562 +0x318\ngithub.com/UnUniFi/chain/app.(*App).BeginBlocker(_, {{0x4a9e658, 0xc000056100}, {0x4ab2eb0, 0xc00004d4c0}, {{0xb, 0x0}, {0xc000bdfdf0, 0x4}, 0x1, ...}, ...}, ...)\n\t/home/user/github.com/UnUniFi/chain/app/app.go:1242 +0xb8\ngithub.com/cosmos/cosmos-sdk/baseapp.(*BaseApp).BeginBlock(_, {{0xc0012f6c60, 0x20, 0x20}, {{0xb, 0x0}, {0xc000bdfdf0, 0x4}, 0x1, {0x397b797c, ...}, ...}, ...})\n\t/home/user/go/pkg/mod/github.com/ununifi/cosmos-sdk@v0.47.1-bank-rc2/baseapp/abci.go:194 +0x878\ngithub.com/cometbft/cometbft/abci/client.(*localClient).BeginBlockSync(_, {{0xc0012f6c60, 0x20, 0x20}, {{0xb, 0x0}, {0xc000bdfdf0, 0x4}, 0x1, {0x397b797c, ...}, ...}, ...})\n\t/home/user/go/pkg/mod/github.com/cometbft/cometbft@v0.37.0/abci/client/local_client.go:283 +0x135\ngithub.com/cometbft/cometbft/proxy.(*appConnConsensus).BeginBlockSync(_, {{0xc0012f6c60, 0x20, 0x20}, {{0xb, 0x0}, {0xc000bdfdf0, 0x4}, 0x1, {0x397b797c, ...}, ...}, ...})\n\t/home/user/go/pkg/mod/github.com/cometbft/cometbft@v0.37.0/proxy/app_conn.go:101 +0x235\ngithub.com/cometbft/cometbft/state.execBlockOnProxyApp({0x4a9e930, 0xc00031f340}, {0x4aafb80, 0xc0005fa168}, 0xc000bb8780, {0x4ab38e8, 0xc0009e5308}, 0x1)\n\t/home/user/go/pkg/mod/github.com/cometbft/cometbft@v0.37.0/state/execution.go:363 +0x4f7\ngithub.com/cometbft/cometbft/state.(*BlockExecutor).ApplyBlock(_, {{{0xb, 0x0}, {0xc000bdf048, 0x6}}, {0xc000bdf080, 0x4}, 0x1, 0x0, {{0x0, ...}, ...}, ...}, ...)\n\t/home/user/go/pkg/mod/github.com/cometbft/cometbft@v0.37.0/state/execution.go:197 +0x285\ngithub.com/cometbft/cometbft/consensus.(*State).finalizeCommit(0xc000d62a80, 0x1)\n\t/home/user/go/pkg/mod/github.com/cometbft/cometbft@v0.37.0/consensus/state.go:1700 +0xf0f\ngithub.com/cometbft/cometbft/consensus.(*State).tryFinalizeCommit(0xc000d62a80, 0x1)\n\t/home/user/go/pkg/mod/github.com/cometbft/cometbft@v0.37.0/consensus/state.go:1609 +0x346\ngithub.com/cometbft/cometbft/consensus.(*State).enterCommit.func1()\n\t/home/user/go/pkg/mod/github.com/cometbft/cometbft@v0.37.0/consensus/state.go:1544 +0xc9\ngithub.com/cometbft/cometbft/consensus.(*State).enterCommit(0xc000d62a80, 0x1, 0x0)\n\t/home/user/go/pkg/mod/github.com/cometbft/cometbft@v0.37.0/consensus/state.go:1582 +0xf29\ngithub.com/cometbft/cometbft/consensus.(*State).addVote(0xc000d62a80, 0xc0012d80a0, {0x0, 0x0})\n\t/home/user/go/pkg/mod/github.com/cometbft/cometbft@v0.37.0/consensus/state.go:2213 +0x241b\ngithub.com/cometbft/cometbft/consensus.(*State).tryAddVote(0xc000d62a80, 0xc0012d80a0, {0x0, 0x0})\n\t/home/user/go/pkg/mod/github.com/cometbft/cometbft@v0.37.0/consensus/state.go:2002 +0x85\ngithub.com/cometbft/cometbft/consensus.(*State).handleMsg(0xc000d62a80, {{0x4a722a0, 0xc0012da020}, {0x0, 0x0}})\n\t/home/user/go/pkg/mod/github.com/cometbft/cometbft@v0.37.0/consensus/state.go:861 +0x656\ngithub.com/cometbft/cometbft/consensus.(*State).receiveRoutine(0xc000d62a80, 0x0)\n\t/home/user/go/pkg/mod/github.com/cometbft/cometbft@v0.37.0/consensus/state.go:788 +0x79a\ncreated by github.com/cometbft/cometbft/consensus.(*State).OnStart\n\t/home/user/go/pkg/mod/github.com/cometbft/cometbft@v0.37.0/consensus/state.go:379 +0x272\n"
```

**Comment by Senna46** [comment]
Fixed the build problem
So, I'll test following functions

- [ ] Vote
- [ ] Delegate
- [ ] Send
- [ ] Derivatives
- [ ] NFT Backed Loan

- [ ] IYA (need to prepare a Strategy)

File: app/app.go [hunk 1]
```diff
@@ -120,23 +128,46 @@ import (
...
+	stakeibcmodule "github.com/UnUniFi/chain/x/stakeibc"
+	stakeibcmodulekeeper "github.com/UnUniFi/chain/x/stakeibc/keeper"
+	stakeibcmoduletypes "github.com/UnUniFi/chain/x/stakeibc/types"
+	yieldaggregator "github.com/UnUniFi/chain/x/yield-aggregator"
+	yieldaggregatorkeeper "github.com/UnUniFi/chain/x/yield-aggregator/keeper"
+	yieldaggregatortypes "github.com/UnUniFi/chain/x/yield-aggregator/types"
+
+	// "github.com/UnUniFi/chain/x/yieldfarm"
+	// yieldfarmkeeper "github.com/UnUniFi/chain/x/yieldfarm/keeper"
+	// yieldfarmtypes "github.com/UnUniFi/chain/x/yieldfarm/types"
```
Comment (mkXultra) [inline app/app.go:156]: plz delete it.
it's not exists.

**Review by mkXultra — CHANGES_REQUESTED** [review]

File: app/app.go (hunk 1 above)
Comment (Senna46) [reply app/app.go]: Deleted all `yieldfarm` comment-out.

**Pull request merged**